resources/seccomp/aarch64-unknown-linux-musl.json

```diff
@@ -191,27 +191,27 @@
             },
             {
                 "syscall": "mmap",
-                "comment": "Used for large buffers sent to api_server",
+                "comment": "Used for reading the timezone in LocalTime::now()",
                 "args": [
                     {
                         "index": 3,
                         "type": "dword",
                         "op": "eq",
-                        "val": 34,
-                        "comment": " libc::MAP_ANONYMOUS | libc::MAP_PRIVATE"
+                        "val": 1,
+                        "comment": "libc::MAP_SHARED"
```

Comment:
is the comment correct? MAP_SHARED being needed for reading the timezone?

Reply:
yes it is. I've checked with strace and in the musl implementation of `localtime_r`
It first checks for the TZ environment variable. If that's not available and if the `/etc/localtime` file is present, it opens the file and mmaps it.